### PR TITLE
fix namespace ignoring when console is invoked

### PR DESCRIPTION
**Pull request by lukyer**
./codecept console accept did not work when namespace attribute occured in codeception.yml file. Helper classes could not be in namespaces.

### CODE REVIEW

File: src/Codeception/Command/Console.php [hunk 1]
```diff
@@ -81,6 +81,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $this->suite    = $suiteManager->getSuite();
 
         $scenario = new Scenario($this->test);
+        if (isset($config["namespace"])) $settings['class_name'] = $config["namespace"].'\\'.$settings['class_name'];
```
Comment (DavertMik) [inline src/Codeception/Command/Console.php]: PSR-2 please

File: src/Codeception/Command/Console.php [hunk 2]
```diff
@@ -81,6 +81,7 @@ public function execute(InputInterface $input, OutputInterface $output)
...
         $actor      = $settings['class_name'];
```
Comment (DavertMik) [inline src/Codeception/Command/Console.php]: Maybe the `$actor` vat can be changed instead of `$settings['class_name']`?

Comment (lukyer) [inline src/Codeception/Command/Console.php]: Don't think so, class name includes a namespace. Property is unusable without the namespace.

**Comment by DavertMik** [comment]
thanks

**Pull request merged**